src/main/java/org/uast/uast/generated/tree/green/Addition.java

```diff
@@ -0,0 +1,262 @@
+ */
+
+package org.uast.uast.generated.tree.green;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.uast.uast.base.Builder;
+import org.uast.uast.base.ChildDescriptor;
+import org.uast.uast.base.ChildrenMapper;
+import org.uast.uast.base.EmptyFragment;
+import org.uast.uast.base.Fragment;
+import org.uast.uast.base.Node;
+import org.uast.uast.base.Type;
+
+/**
+ * Node that describes the 'Addition' type.
+ *
+ * @since 1.0
+ */
+public final class Addition implements BinaryExpression {
+    /**
+     * The type.
+     */
+    public static final Type TYPE = new TypeImpl();
+
+    /**
+     * The number of children.
+     */
+    private static final int CHILD_COUNT = 2;
+
+    /**
+     * The fragment associated with the node.
+     */
+    private Fragment fragment;
+
+    /**
+     * List of child nodes.
+     */
+    private List<Node> children;
+
+    /**
+     * Child with the 'left' tag.
+     */
+    private Expression left;
+
+    /**
+     * Child with the 'right' tag.
+     */
+    private Expression right;
+
+    /**
+     * Constructor.
+     */
+    private Addition() {
+    }
+
+    @Override
+    public Type getType() {
+        return Addition.TYPE;
+    }
+
+    @Override
+    public Fragment getFragment() {
+        return this.fragment;
+    }
+
+    @Override
+    public String getData() {
+        return "";
+    }
+
+    @Override
+    public int getChildCount() {
+        return Addition.CHILD_COUNT;
+    }
+
+    @Override
+    public Node getChild(final int index) {
+        return this.children.get(index);
+    }
+
+    @Override
+    public Expression getLeft() {
+        return this.left;
+    }
+
+    @Override
+    public Expression getRight() {
+        return this.right;
+    }
+
+    /**
+     * Type descriptor of the 'Addition' node.
+     *
+     * @since 1.0
+     */
+    private static class TypeImpl implements Type {
+        /**
+         * The 'Addition' string.
+         */
+        private static final String ADDITION = "Addition";
+
+        /**
+         * The 'Expression' string.
+         */
+        private static final String EXPRESSION = "Expression";
+
+        /**
+         * The list of child types.
+         */
+        private static final List<ChildDescriptor> CHILDREN =
+            Collections.unmodifiableList(
+                Arrays.asList(
+                    new ChildDescriptor(
+                        TypeImpl.EXPRESSION,
+                        false
+                    ),
+                    new ChildDescriptor(
+                        TypeImpl.EXPRESSION,
+                        false
+                    )
+                )
+            );
+
+        /**
+         * The 'BinaryExpression' string.
+         */
+        private static final String BINARY_EXPRESSIO = "BinaryExpression";
```

Comment:
You have lost the last char

Reply:
The name of such constants (static private final fields) cannot exceed 16 characters. The name has been truncated to prevent Qulice  warning. This is an automatic conversion, the simplest truncating algorithm has been applied.